static/docs/get-started/install.md

```diff
@@ -8,6 +8,12 @@ To install DVC from terminal, run:
 ```dvc
 $ pip install dvc
 ```
+### Expand to install from Anaconda repository
+To install DVC from Anaconda prompt, run:
+
+```dvc
+$ conda install -c derickl dvc
```

Comment:
@efiop do you know who made this project - https://anaconda.org/derickl/dvc?
@ryokugyu the version there is really outdated.

Reply:
it may be outdated, but it's the latest available on Anaconda repository.
Any chance of updating the repository soon?

Reply:
@shcheklein No, I don't.

@ryokugyu We will need to contact that guy and see if he would be willing to maintain it. Otherwise, we'll have to setup our own conda repo and maintain everything ourselves.

Reply:
Did anyone try to contact the guy? Else I will try and let you people update about it?

Reply:
@ryokugyu the problem is that we don't know exactly who this guy is and how can we contact him

Reply:
@shcheklein any update on this?

Reply:
@ryokugyu no, the status is the same :(